packages/xo-web/src/xo-app/backup-ng/new/utils.js

```diff
@@ -38,3 +39,35 @@ Number.propTypes = {
   onChange: PropTypes.func.isRequired,
   value: PropTypes.number.isRequired,
 }
+
+export const FormFeedback = ({
+  showError,
+  error,
+  component: Component,
+  ...props
+}) => (
+  <div>
+    <Component
+      {...props}
+      style={
+        showError === undefined
+          ? undefined
+          : {
+              borderColor: showError ? 'red' : 'green',
+              ...props.style,
+            }
+      }
+    />
+    {showError && (
+      <span className='text-danger'>
+        <Icon icon='alarm' /> {error}
+      </span>
+    )}
+  </div>
+)
+
+FormFeedback.propTypes = {
+  component: PropTypes.node.isRequired,
+  error: PropTypes.node.isRequired,
+  showError: PropTypes.bool,
```

Comment:
I'd say `error` should be the boolean and `message` should be the feedback node.